.rhcicd/clowdapp-engine.yaml

```diff
@@ -302,6 +302,9 @@ parameters:
 - name: RECIPIENT_PROVIDER_USE_IT_IMPL
   description: Temporary recipients retrieval switch between RBAC and IT
   value: "false"
+- name: SENTRY_DSN
+  description: The DSN to push data to Sentry
+  value: https://username@credential.ingest.sentry.io/projectNumber?environment=
```

Comment:
I've checked and it doesn't fail, no. My idea behind adding that `value` was to provide an example where we explicitly stress the `?environment=` part, since I see it could be easy to forget about :stuck_out_tongue_closed_eyes: 

But I wouldn't mind removing it

Reply:
Then maybe this is a better place for the example:
```
  description: The DSN to push data to Sentry (i.e. https://public_key@host/project_id?environment=)
```

Reply:
You're right, yeah. Moved it! Thanks!